### PR TITLE
change getWorkspaceAccessLevels to use only realmACLs

### PR DESCRIPTION
**Pull request by dvoet**
https://broadinstitute.atlassian.net/browse/GAWB-289
# 
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] Tell  that the PR exists if he wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [ ] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x]  sign off
- [x] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Squash commits, rebase if necessary
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Merge to develop 
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: Check configuration files in Jenkins in case they need changes
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAO.scala [hunk 1]
```diff
@@ -89,28 +89,30 @@ class GraphAuthDAO extends AuthDAO with GraphDAO {
       // user -> group -> ...some other groups... -> group -> workspace access map
       //Multiple paths implies that the user belongs to multiple groups associated with the workspace.
       val accessMapPaths =
-        userPipeline(db, user).as("vtx").in()
+        userPipeline(db, user).as("vtx").inE().outV()
```
Comment (dvoet) [inline src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphAuthDAO.scala:124]: did this to capture the edges between workspace and acl maps, acl maps and group more effectively

**Comment by dvoet** [comment]
Note that there are 2 failing unit tests because we are not yet creating the intersection group membership. These should be fixed by GAWB-295.

**Comment by jmthibault79** [comment]
I'm seeing two failed unit tests, but I don't think they are the right ones:

```
[info] - should clone a workspace's Realm if it exists *** FAILED ***
[info]   Expected 201 Created, but got 404 Not Found (WorkspaceApiServiceSpec.scala:472)
[info] - should not allow changing a workspace's Realm if it exists *** FAILED ***
[info]   Expected 422 Unprocessable Entity, but got 404 Not Found (WorkspaceApiServiceSpec.scala:510)
```

Why would this PR cause cloning to break?

**Comment by jmthibault79** [comment]
Looks good otherwise but I am very confused about the tests.

**Comment by dvoet** [comment]
used talkings but for posterity, the cloning unit tests break because the workspace being cloned is created without correctly populated realmACLs groups so the user does not have access to those workspaces leading to 404 errors.

**Comment by jmthibault79** [comment]
:+1:

**Pull request merged**